src/models/users.js

```diff
@@ -13,9 +13,10 @@ const usersSchema = mongoose.Schema({
   email: String,
```

Comment:
Up here in the file, you have userId and displayName set as require: true, which is a slip in the boilerplate, the keyword should be `required`. Please change that.